lib/csv/parser.rb

```diff
@@ -1197,20 +1197,12 @@ def parse_row_end
     end
 
     def strip_value(value)
-      return value unless @strip
-      return nil if value.nil?
+      return value if not @strip or value.nil?
 
       case @strip
       when String
-        size = value.size
-        while value.start_with?(@strip)
-          size -= 1
-          value = value[1, size]
-        end
-        while value.end_with?(@strip)
-          size -= 1
-          value = value[0, size]
-        end
+        nil while value.delete_prefix!(@strip)
```

Comment:
I don't think that this is readable.
How about the following?

```suggestion
        while value.delete_prefix!(@strip)
          # do nothing
        end
```

Reply:
I just did not want to have an empty while block. I will do the suggested changes.